pkgs/development/libraries/libredwg/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, stdenv, fetchFromGitHub, autoreconfHook, pkgconfig, texinfo, pcre2
+, enablePython ? false, python, swig, libxml2, ncurses
+}:
+let
+  isPython3 = enablePython && python.pythonAtLeast "3";
+in
+stdenv.mkDerivation rec {
+  pname = "libredwg";
+  version = "0.10.1";
+
+  src = fetchFromGitHub {
+    owner = "LibreDWG";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256:1zd721z2nriw1jlrh4y1fj59b0dnymhd4kwp8rqw16bs84gda37n";
+  };
+
+  nativeBuildInputs = [ autoreconfHook pkgconfig texinfo ];
+
+  buildInputs = [ pcre2 ]
+    ++ lib.optionals enablePython [ python swig libxml2 libxml2.dev ]
```

Comment:
Also there is no need to specify dev output explicitly, it will be added from `libxml2` already.

Reply:
I thought so, too and the 0.9. version of the PR did not need that.
But after rebasing it did not find libxml during check phase anymore.

Git bisect pointed to 6a97422a7555af7db66cd26cdafaff6aad5eb7c8 as the cause. Explicitly adding the dev output fixed it.

Reply:
Oh, right, you are replacing `libxml2` with a Python package. Is that needed? I would not expect swig generated bindings to interface with python libxml2 module. Maybe add a comment with an explanation since it is unexpected.